Week3/js-exercises/ex5-LotteryMachine.js

```diff
@@ -25,14 +25,40 @@ Don't you just love the thrill of the lottery? What if I told you we can make ou
  if the array value is divisible by both 3 and 5.
 
 */
-
-function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
-  const numbers = [];
+debugger;
+function threeFive(startNumber, stopNumber, threeCallback, fiveCallback) {
   // make array
-  // start at beginning of array and check if you should call threeCallback or fiveCallback or go on to next
+  function makeArray() {
+    let numbers = [];
+
+    for (let i = 0; i <= stopNumber - startNumber; i++) {
```

Comment:
This works great! What do you think of this slight modification? 
```suggestion
    for (let i = startNumber; i <= stopNumber ; i++) {
       numbers.push(i)
```
Choose whatever you like best 🙂

Reply:
Love it ! ...much more elegant ! Thank you!